src/data/data.py

```diff
@@ -268,7 +268,7 @@ def train_dataloader(self):
     def val_dataloader(self):
         loader = DataLoader(self.val_dataset,
                             batch_size=self.batch_size,
-                            shuffle=False,
+                            shuffle=True,
```

Comment:
we def shouldnt set shuffle to true in val dataloader

Reply:
i tried to do this by picking a random batch id in the other pr but its def not advised to shuffle val batches - any other ideas? :)

Reply:
random batch id is an alternative. are you worried about floating point errors when doing val shuffling?

Reply:
its just not good practice to shuffle your val dataset - we want this to be consistent across different models etc. i actually think if you run this there is a warning in lightning tell you not to do this

Reply:
fair enough :) will switch to a random batch.